src/e2e/java/teammates/e2e/cases/e2e/AdminSearchPageE2ETest.java

```diff
@@ -177,17 +178,17 @@ private void verifyInstructorRowContent(InstructorAttributes instructor, Account
     }
 
     private String getExpectedInstructorHomePageLink(InstructorAttributes instructor) {
-        return instructor.isRegistered() ? createUrl(Const.WebPageURIs.INSTRUCTOR_HOME_PAGE)
-                .withUserId(instructor.googleId)
-                .toAbsoluteString()
-                : "";
+        String googleId = instructor.isRegistered() ? instructor.googleId : "undefined";
```

Comment:
Can we use empty string instead of `undefined`? Why the `undefined` is there in the first place?

Reply:
Yep, we can, we can even remove the param if its value its an empty string, which is what i have instead opted to do